edenai_apis/llmengine/clients/litellm_client/litellm_client.py

```
@@ -914,6 +914,99 @@ async def generate_chunks():
+            try:
+                cache_kwargs = {}
+                for key in ("cache_creation_input_tokens", "cache_read_input_tokens"):
+                    if key in usage:
+                        cache_kwargs[key] = usage[key]
+                cost = completion_cost(
+                    model=model_name,
+                    prompt_tokens=usage.get("input_tokens", 0),
+                    completion_tokens=usage.get("output_tokens", 0),
+                    **cache_kwargs,
+                )
+            except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)